### PR TITLE
#4:update style detail blog html file

### PR DESCRIPTION
**Pull request by Lanng21**
image
image
image
image
image

### CODE REVIEW

**Comment by haunguyen15-tt** [comment]
No new line at the end in all files

File: src/components/blog-details-comment.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<div class="blog-details-comment-wrapper mt-5">
+    <figure class="blog-details-comment-wrapper-thumb">
+        <img class="blog-details-comment-wrapper-thumb__fit-image img-fluid rounded-circle mr-3 mb-3"
```
Comment (baonguyen26) [inline src/components/blog-details-comment.html:3]: improve?

File: src/components/detail.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                <li>By: <a href="{{ linkinfo }}">{{ info }}</a> <span>{{ day }}</span></li>
+                <li>{{ numbercomment }}</li>
+            </ul>
+        </div>
+        <!-- Blog details title & Meta End -->
+
+        <!-- Content Start -->
+        <p>{{ content }}</p>
+        <!-- Blogquote Start -->
+        <blockquote> {{ blogquote }}</blockquote>
```
Comment (baonguyen26) [inline src/components/detail.html:26]: blogQuote

File: src/components/detail.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        <!-- Content End -->
+
+        <!-- Blog Details Tags & Social Shear Start -->
+        <div class="blog-details-content__tagsocial mb-n4">
+
+            <!-- Blog Details Tags Start -->
+            <div class="blog-details-content__tagsocial-tag mb-4">
+                <ul>
+                    <li><a href="">breakfast</a></li>
+                    <li><a href="">dinner</a></li>
```
Comment (baonguyen26) [inline src/components/detail.html:38]: `javascript: void(0)`

**Pull request closed without merging**